smooth/components/component_storage_h2.py

```diff
@@ -40,6 +38,8 @@ def __init__(self, params):
 
         # ------------------- UPDATE PARAMETER DEFAULT VALUES -------------------
         self.set_parameters(params)
+        # Initial storage level [kg].
+        self.storage_level_init = 0.5 * self.storage_capacity
```

Comment:
This makes it impossible to set another init level than half of the storage capacity. Is that what you wanted?

Reply:
Ah yeah that was to test it, think I've sorted it now ! Thanks